Signaling/app/src/main/java/com/tokbox/android/tutorials/signaling/ChatMessage.java

```diff
@@ -0,0 +1,33 @@
+package com.tokbox.android.tutorials.signaling;
+
+public class ChatMessage {
+
+    private String mMessageText;
+    private Boolean mRemote;
+
+    public ChatMessage(String messageText) {
+        mMessageText = messageText;
+        mRemote = false;
+    }
+
+    public static ChatMessage fromData(String messageData) {
+        return new ChatMessage(messageData);
+    }
+
+    public String getMessageText() {
+        return mMessageText;
+    }
+
+    public Boolean getRemote() {
+        return mRemote;
+    }
+
+    public void setRemote(Boolean remote) {
+        mRemote = remote;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
I believe this is because we are using an ArrayAdapter. From [Android's ArrayAdapter Docs](https://developer.android.com/reference/android/widget/ArrayAdapter.html):

> By default this class expects that the provided resource id references a single TextView....
> 
> ...However the TextView is referenced, it will be filled with the toString() of each object in the array. You can add lists or arrays of custom objects. Override the toString() method of your objects to determine what text will be displayed for the item in the list.